client/extensions/woocommerce/app/settings/shipping/shipping-classes.scss

```diff
@@ -0,0 +1,185 @@
+/** @format */
+
+/**
+ * Shipping classes list/table
+ */
+
+.shipping__classes {
+	padding: 0;
+}
+
+.shipping__classes-row {
```

Comment:
It looks like the styling rules for the shipping class list view is essentially is duplicated from the shipping zone list view. Are there reasons against generalizing and re-using much of the latter's CSS?

Reply:
The only reason was that I thought I should use the `shipping__classes` prefix within the shipping classes list. That happened because I was confused by an eariler message from the git pre-commit hook. Now it is done that way.